Project/Engine/src/sysFrame/System/Window/Window.cpp

```
@@ -12,13 +18,57 @@ LRESULT CALLBACK Window::WindowProc(HWND hWnd, UINT msg, WPARAM wParam, LPARAM l
 	case WM_DESTROY:
 		PostQuitMessage(0);
-		return 0;
+	/*case WM_PAINT:
+		DisplayLogo(hWnd);
+		return 0;*/
 	}
 
 	return DefWindowProc(hWnd, msg, wParam, lParam);
 }
 
+void Window::DisplayLogo(HWND hWnd) {
```

cppcheck (2.10-2)

[style] The function 'DisplayLogo' is never used.

(unusedFunction)